datm/cime_config/buildnml

```diff
@@ -73,11 +73,13 @@ def _create_namelists(case, confdir, inst_string, infile, nmlgen, data_list_path
     config['datm_mode'] = datm_mode
     config['datm_co2_tseries'] = datm_co2_tseries
     config['datm_presaero'] = datm_presaero
-    config['cime_model'] = get_model()
-    if case.get_value('PTS_MODE'):
-        config['mode'] = 'single_column'
+
+    scol_lon = float(case.get_value('PTS_LON'))
```

Comment:
I think that this should be:
```
if case.get_value('PTS_LON'):
   scol_lon = float(case.get_value('PTS_LON'))
```
same with scol_lat and ATM_DOMAIN_FILE

Reply:
Right!